### ISSUE TITLE
information on media edit screen

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#293]
it would be useful to list information on the media edit screen including who uploaded the item & when, & details of the file such as dimensions, filesize, file format, filename, & a download link.

**Comment by MaxButchart** [comment]
Where on the media edit page should this information be displayed? Is there any particular format this information should be displayed in?

**Comment by helenvarley** [comment]
please use the current UpStage workshop layout as a guideline (i've said before that the image displayed on the media edit page doesn't need to be huge - at the moment on Limelight media edit pages there is a massive amount of space wasted by a larger-than-lifesize image of the media item)
![screenshot at 2018-05-02 11 28 14](https://user-images.githubusercontent.com/3437324/39516041-09ab7b00-4dfc-11e8-933c-9e7a112e64a2.png)

**Comment by MaxButchart** [comment]
![Uploading current version.png…]()
This is what I have so far, i'm currently waiting on some code written by Thong to be merged to get the info about the user uploading the media. I have the stages as buttons that link to the stage page, like in the media edit page, would you like the stages to be plain text instead? Any other formatting changes?

**Comment by helenvarley** [comment]
should the tags also be listed here, in the same way as the stages? (as buttons that would then connect to a list for that tag)

**Comment by charlie0605** [comment]
@helenvarley so you would like the tag to be listed there too, under the stage? Does that mean that the Stage and the Tag at the top could go or you also want them to stay to have more choice of access?

**Comment by vickis** [comment]
yes agreed :)

**Comment by MaxButchart** [comment]
@rohrlach I've noticed working on this issue that there's a lot of duplicate code for each different media type, should I look at refactoring it?

**Comment by rohrlach** [comment]
Yes @MaxButchart, you absolutely should refactor if you see an opportunity
